spring-pulsar/src/test/java/org/springframework/pulsar/core/CachingPulsarProducerFactoryTests.java

```diff
@@ -74,7 +75,7 @@ void cleanupFromTests() {
 	}
 
 	@Test
-	void createProducerMultipleCalls() throws PulsarClientException {
```

Comment:
**[UNRELATED]** Leftover from removal of checked exception in Spring Pulsar APIs.